static/app/components/releaseSidebarSection.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+
+import space from 'app/styles/space';
+
+type Props = {
+  title: React.ReactNode;
+  children?: React.ReactNode;
+  icon?: React.ReactNode;
+};
+
+/**
+ * Used to add a new sidebar section on Release Details page.
+ */
+function ReleaseSidebarSection({title, children, icon, ...props}: Props) {
```

Comment:
The design of sidebar sections and titles is used all over the app (transaction summary, project details, alert rule, ..) not just release details. Should we make something reusable so that we can use it in those places too?
I think this refactor should be done with the whole app in mind, not just release details.
cc @vuluongj20

Reply:
For sure, I can update the naming here - I mostly made it to where it should work on any sidebar in the app, but I only updated the Release Details page. Should I start working on updating other pages with this after?

Reply:
Renaming sounds like a good idea 👍